intl/msg_hash_us.h

```diff
@@ -6577,6 +6777,10 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_ACCOUNTS_RETRO_ACHIEVEMENTS,
    "Earn achievements in classic games. For more information, visit 'https://retroachievements.org'."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_ACCOUNTS_RETRO_ACHIEVEMENTS,
+   "Login details for your Retro Achievements account. Visit retroachievements.org and sign up for a free account.\nAfter you are done registering, you need to input the username and password into RetroArch."
```

Comment:
Bad inheritance here: It should read "RetroAchievements", as a single word. https://retroachievements.org/

Reply:
Noted, will fix in phase 2 PR.